proto/options.proto

```diff
@@ -227,4 +227,5 @@ message Options {
   optional bool linear_references = 45;                                   // Include linear references for graph edges returned in certain responses.
   repeated CostingOptions recostings = 46;                                // Costing options to use to recost a path after it has been found
   repeated Ring exclude_polygons = 47;                                    // Rings/polygons to exclude entire areas during path finding
+  optional bool invariant_postprocess = 48 [default = true];              // Use invariant time after a route is built, on recosting or leg building stages.
```

Comment:
this is so strange to me. is there something you dont like about my suggestion to avoid misusing the word `invariant`? from this i understand you want to keep using the `invarient` date time type but then you want recosting to ignore it and use the `depart_at` type. i think it would be way less confusing for the api to just allow a `date_time.type` of  `depart_at/current` (or even `arrive_by` once we have `recost_reverse`) to go to the bidirectional algorithm, where it can just not vary the time (for now) while finding the path and then use either the `depart_at` time or `current` time in the recosting part as specified in the request.

let me know if you have some time to talk this over. i think hijacking `invariant` to do what you want is poor API design. It would be easier if you let the requester say they want to use date_time in bidirectional astar. there is another option that i dont like which would be to add a second date_time parameter to be used in recosting only. but its a bad option as the recosting that already exists is complicated enough.

maybe im missing something and it has to be the way you've laid out, but i cant see the reason why. do let me know when you have the chance.

Reply:
I got what you mean by `force_bidirectional` sounds like a good idea. The second with additional parameter will make the code complicated and messy, agree with you. I'll implement the solution with `force_bidirectional` on Wednesday, OOO now.

Reply:
I've implemented a draft version without tests for now. Bidirectional algo will use invariant time for all date_time_types untill we find a good implementation that works with time updates. @kevinkreiser What do you think about it now? Recosting uses invariant only when it was specified in date_time_type.

Reply:
I use `prioritize_bidirectional`, because bidirectional a* has problems on short routes, when origin and destination edges are connected. Don't want to call it in such cases